docs/source/advanced_usage.rst

```diff
@@ -52,15 +52,18 @@ concurrent context.
 
 Below is an example that demonstrates how to use pyQuil in a multithreading scenario:
 
+..
+    TODO: This should be covered by doctests but hangs locally.
+
```

Comment:
Do we intend to resolve this? IMO it's okay to leave out things which would otherwise be a rabbit hole, best to not let the perfect be the enemy of the merge

Reply:
I vaguely remember we discussed not doing this offline. I'll remove this for now.